js/team.js

```diff
@@ -0,0 +1,123 @@
+// Populate modal
```

Comment:
This file fetches all of the team members from the CMS, generates card components for each, and generates modal data for each.